src/main/java/hudson/plugins/jira/RunScmChangeExtractor.java

```diff
@@ -25,7 +26,8 @@ private RunScmChangeExtractor() {
 
     public static List<ChangeLogSet<? extends Entry>> getChanges(Run<?, ?> run) {
         if (run instanceof AbstractBuild) {
-            return ((AbstractBuild<?, ?>) run).getChangeSets();
```

Comment:
inlined restricted method, pending core update to IIRC 2.60.x when you can use the official API

Reply:
Is this related to the POM fixes?

Reply:
Yes. When you update the POM, your plugin (correctly) fails to compile. See https://github.com/jenkinsci/plugin-pom/pull/61.